python/tskit/trees.py

```diff
@@ -1608,6 +1615,24 @@ def draw_svg(
         :param bool force_root_branch: If ``True`` always plot a branch (edge) above the
             root(s) in the tree. If ``None`` (default) then only plot such root branches
             if there is a mutation above a root of the tree.
+        :param float symbol_size: Change the default size of the node and mutation
+            plotting symbols. If ``None`` (default) use a standard size.
+        :param bool x_axis: Should the plot have an X axis line, showing the start and
+            end position of this tree along the genome. If ``None`` (default) do not
+            plot an X axis.
+        :param bool y_axis: Should the plot have an Y axis line, showing time (or
+            ranked node time if ``tree_height_scale="rank"``). If ``None`` (default)
+            do not plot a Y axis.
+        :param str x_label: Place a label under the plot. If ``None`` (default) and
+            there is an X axis, create and place an appropriate label.
+        :param str y_label: Place a label to the left of the plot. If ``None`` (default)
+            and there is a Y axis,  create and place an appropriate label.
+        :param list y_ticks: A list of Y values at which to plot tickmarks (``[]``
+            gives no tickmarks). If ``None``, plot one tickmark for each unique
+            node value.
+        :param bool y_gridlines: Whether to plot horizontal lines behind the tree
+            at each y tickmark. If ``None`` (default), only plot gridlines if a list
```

Comment:
I think y_gridlines should be independent of y_ticks. It would be annoying to have to turn off y_gridlines if you just wanted specific ticks.

Reply:
Oh yes, I simply forgot to change the documentation here. Thanks.